download/download_test.go

```diff
@@ -109,3 +109,39 @@ func (suite *TestSuite) TestCreateFilePath() {
 	err = os.RemoveAll("one")
 	assert.NoError(suite.T(), err)
 }
+
+func (suite *TestSuite) TestGetURLsListFile() {
+
+	currentPath, err := os.Getwd()
+	assert.NoError(suite.T(), err)
+
+	// Folder URL does not exist
+	fileLocation := "https://some/base/A352744B-2CB4-4738-B6B5-BA55D25FB469/some/"
+
+	urlsFilePath, err := GetURLsListFile(currentPath, fileLocation)
+	assert.Equal(suite.T(), urlsFilePath, "")
+	// The error differs locally and in the repo, therefore checking that error exists
+	// In github the error produced is
+	// assert.EqualError(suite.T(), err, "failed to download file, reason: Get \"https://some/base/A352744B-2CB4-4738-B6B5-BA55D25FB469/some/urls_list.txt\": dial tcp: lookup some: Temporary failure in name resolution")
+	// Locally the error varies with the following being one alternative
+	// assert.EqualError(suite.T(), err, "failed to download file, reason: Get \"https://some/base/A352744B-2CB4-4738-B6B5-BA55D25FB469/some/urls_list.txt\": dial tcp: lookup some: no such host")
+	assert.NotEqual(suite.T(), err, nil)
```

Comment:
If you want to be fancy and have a test that will pass you can use the `net/http/httptest` package